primitives/src/pcs/univariate_kzg/srs.rs

```diff
@@ -155,10 +157,18 @@ mod tests {
         let h = h.into_affine();
         let beta_h = (h * beta).into_affine();
 
+        let powers_of_h_threshold = 10;
```

Comment:
You don't want this to be `max_degree`?

Reply:
It can be an API parameter after [70f0a0a](https://github.com/EspressoSystems/jellyfish/pull/406/commits/70f0a0a94e5c213707b889d1302a5eb0243dbaf8) and [7ad9865](https://github.com/EspressoSystems/jellyfish/pull/406/commits/7ad98656d4f843c61522392d0313624e957fe4a1)